Session-4/CurrencyExchangeApp/CurrencyConverter/CurrencyConverter.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Configuration;
+using RestSharp;
+using System.Collections.Generic;
+using System.IO;
+
+
+namespace CurrencyConverter
+{
+    public class CurrencyExchange
+    {
+        public void GetApidata()
+        {
+            try {
+                RestClient client = new RestClient("https://api.apilayer.com/currency_data/live?source=INR&currencies=");
+                RestRequest request = new RestRequest();
+                request.AddHeader("apikey", "NKJlyNE9umLraoNvP87N7W9v6NNePbI8");
+                ApiResponse apiResponse = JsonConvert.DeserializeObject<ApiResponse>(client.Get(request).Content);
+                apiResponse.quotes.Add("timestamp", apiResponse.timestamp);
+                string path = System.Configuration.ConfigurationManager.AppSettings["path"];
+                File.WriteAllText(path, JsonConvert.SerializeObject((object)apiResponse.quotes));
+            }
+            catch(Exception e) {
+                throw e.InnerException;
+            }
+                
+        }
+
+        public double Convert(string fromCurrency, string toCurrency, double Currencyvalue)
+        {
+            CurrrencyNotFoundException notFoundException = new CurrrencyNotFoundException("Enter a valid Currency[Ex : USD, INR]");
+            if (toCurrency == fromCurrency)
+                return 1.0;
+            string key = fromCurrency + toCurrency;
+            if (File.Exists("C:\\Users\\prashanth yarram\\OneDrive\\Desktop\\sample.txt"))
```

Comment:
Keep the file path in config. Also name the file properly (like ExchangeRates.json)